lib/redux/reducers/users.js

```diff
@@ -222,11 +222,14 @@ export function uploadTargetUser(state = null, action) {
     case actionTypes.LOGIN_SUCCESS:
     case actionTypes.SET_USER_INFO_FROM_TOKEN:
       const { user, profile, memberships } = action.payload;
+      const uploadMemberships = _.filter(memberships, (mship) => {
+        return !_.isEmpty(_.get(mship, ['profile', 'patient']));
+      });
```

Comment:
The basic misunderstanding was that the memberships _always_ includes the `loggedInUser`, even if they aren't a PWD! The mistake slipped in when I refactored the `users` reducer to reduce (har har) the nesting...